src/model/formats/abstract-format.ts

```diff
@@ -0,0 +1,66 @@
+import { ChecklistFile } from '../../../gen/ts/checklist';
+
+declare global {
+  const enum FormatId {}
+}
+
+export type FileExtension = string;
```

Comment:
Maybe add a comment saying whether this includes the dot?

Reply:
I think now that it's properly expressed at the type system level in 9fae08a6f0898091639311cc528c80972555f9a5, it's actually become self-documenting.